modules/ducktests/tests/ignitetest/tests/rebalance/__init__.py

```diff
@@ -45,9 +48,7 @@ def preload_data(context, config, backups, cache_count, entry_count, entry_size,
         startup_timeout_sec=timeout)
     app.run()
 
-    return (get_event_time(
-        app, app.nodes[0], "Marking as initialized") - get_event_time(
-        app, app.nodes[0], "Data generation started")).total_seconds()
+    return (app.await_stopped() - app.await_started()).total_seconds()
```

Comment:
I don't understand why "await" methods returns some time.
Why we call `await_started` only after `await_stopped`